_layouts/default.html

```diff
@@ -6,7 +6,7 @@
   <title>{% if page.title %}{{page.title}}{% else %}{{ site.title | escape }}{% endif %}</title>
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <link rel="icon" type="image/png" href="{{ '/images/favicon-32x32.svg' | relative_url }}">
-  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="preload" href="{{ '/assets/fonts/playfair-display.woff2' | relative_url }}" as="font" type="font/woff2" crossorigin>
```

Comment:
I'm skeptical that self hosting the font will make things faster. And this change makes it harder for people to change the fonts. 

I suggest we continue to use Google Fonts cdn as the default, but we include the code to self host the font and have it commented out.

Reply:
Done!